### PR TITLE
Readme no nersc account

### LINKED ISSUE #86: What to do if I do not have a NERSC account
Add descriptions in the README about what to do if I do not have a NERSC account. i.e. how to use [Datalab](https://datalab.noirlab.edu/)

### PR DESCRIPTION
**Pull request by stephjuneau**
I only modified the top-level README file. I added a Table of content, a section on external database access including what Stephen asked about TOPCAT access to the TAP handle. I also added a one-line section on DESI Data Acknowledgments pointing to the Data Doc website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ Welcome to the repository designed for tutorials that guide you through the proc
 * Tutorials that were demonstrated at specific DESI meetings, each segregated under directories marked by the date of the meeting. For example, meetings/Dec2020.
 * Tutorials designed around wide-ranging subjects (like onskydata) or specific packages (such as fiberassign, redrock).
 
+### Table of Content
```
Comment (weaverba137) [inline README.md]: "Contents"

File: README.md [hunk 2]
```diff
@@ -116,6 +123,41 @@ Tutorials hosted in other packages include (but haven't been recently vetted):
...
+```
+>> from sparcl.client import SparclClient 
+>> client = SparclClient()
+```
+
+There are instructions and useful examples in the [How-to-use-SPARCL tutorial notebook](https://github.com/astro-datalab/notebooks-latest/blob/master/04_HowTos/SPARCL/How_to_use_SPARCL.ipynb).
+
+# Acknowledgments
+
+Any use of DESI data whether via NERSC or external databases requires [DESI Data Acknowledgments](https://data.desi.lbl.gov/doc/acknowledgements/).
```
Comment (weaverba137) [inline README.md]: Double-check the spelling: https://github.com/desihub/desidatadocs/blob/edr/docs/acknowledgments.md

**Review by weaverba137 — CHANGES_REQUESTED** [review]
I found a couple of spelling errors.

File: README.md (hunk 2 above)
Comment (sbailey) [reply README.md]: for context, there is a British vs. American spelling difference for "acknowledgments", and even the pubboard acknowledgments wiki page used both on a single page.  Current main uses British acknowledgements (with an extra "e") but the edr branch soon to be merged will switch to the American spelling "acknowledgments" as the final URL to use.  We need to check this on the NOIRLab websites too!

**Review by forero — APPROVED** [review]
Aside from the spelling issue, I believe these extra bits in the README are useful.

File: README.md (hunk 2 above)
Comment (stephjuneau) [reply README.md]: Yes, we also had (have?) both versions of the spelling on the Data Lab website. After figuring out which is the American spelling, we have so far fixed it on the SPARCL website (https://astrosparcl.datalab.noirlab.edu/sparc/acknowledgments/).

That said, we won't be able to change the SPARCL website for a while (complicated -- it's tied to code releases) so are you able to add a re-direct such that "https://data.desi.lbl.gov/doc/acknowledgements/" re-directs to "https://data.desi.lbl.gov/doc/acknowledgments/" meaning that either spelling would not cause an error? (if not, a hack would be to create two identical "acknowledgments.md" and "acknowledgements.md" pages for now, which is of questionable taste but might work in a pinch).

Comment (stephjuneau) [reply README.md]: We have used the Acknowledgments in many places so tracking it all down might take a while even on the Data Lab website (which we can still modify today, unlike the SPARCL website).

Comment (stephjuneau) [reply README.md]: @sbailey  could you confirm that I should change the URL in the README (and everywhere we still possibly can) to "https://data.desi.lbl.gov/doc/acknowledgments/" without the British "e"?

Comment (sbailey) [reply README.md]: yes, please change it everywhere you can to the American spelling https://data.desi.lbl.gov/doc/acknowledgments/ (which does not yet exist).  I'll look into our options for also supporting redirect for the British spelling.

**Comment by forero** [comment]
@stephjuneau Please go ahead and merge this branch whenever you are ready. I will proceed merging the database branch afterwards.

File: README.md (hunk 2 above)
Comment (sbailey) [reply README.md]: https://data.desi.lbl.gov/doc/acknowledgments/ now does exist as the preferred link to use, and desihub/desidatadocs#60 will restore support for the older spelling for pre-deployed sites using the older URL.

File: README.md (hunk 1 above)
Comment (stephjuneau) [reply README.md]: Fixed. Thanks!

File: README.md (hunk 2 above)
Comment (stephjuneau) [reply README.md]: OK great. Marking as resolved.

**Comment by stephjuneau** [comment]
OK so I made the two small changes requested by Ben (removed the extra "e" in acknowledgments URL). I have also just added a new README to the getting_started/ folder. This links to the other two previous .md files but we could incorporate them directly if preferred? @akremin will need to update the link to the EDR paper repo. (@forero what do you think of the new README file? do the links to the other md files work for you?)

**Comment by stephjuneau** [comment]
I was trying to let Ben know that I made his two requested changes but I accidentally re-requested a review. Not sure that was needed but I'll wait to hear a minimum of one approval before merging (whoever sees this first :-) )

File: README.md [hunk 3]
```diff
@@ -116,6 +123,41 @@ Tutorials hosted in other packages include (but haven't been recently vetted):
...
+
+The [SPectra Analysis & Retrievable Catalog Lab (SPARCL)](https://astrosparcl.datalab.noirlab.edu) contains 
+DESI EDR spectra that were coadded per healpix. It is a searchable database that can be used via the Astro Data Lab 
+JupyterLab Notebook. Alternatively, the Python client can be installed locally using:
+```
+pip install sparclclient
+```
+The client can be loaded within a Python session or program via:
+```
+>> from sparcl.client import SparclClient
```
Comment (weaverba137) [inline README.md]: The standard Python prompt has three arrows: `>>>`.

File: getting_started/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Other useful references
+
+### DESI Data Documentation Website
+
+The [DESI Data Documentation](https://data.desi.lbl.gov/doc/) website describes data access, data format, data releases, and includes links to technical papers as well as information on data license and acknowledgments.
+
+### DESI Helpdesk
+
+If you have a question, please visit the [DESI User Forum](https://help.desi.lbl.gov)
```
Comment (weaverba137) [inline getting_started/README.md]: Period at the end of the sentence.

**Comment by stephjuneau** [comment]
Hi @weaverba137 are you OK with me merging this PR? I've made both fixes (and added a README in case you want to see it: https://github.com/desihub/tutorials/blob/readme-no-nersc-account/getting_started/README.md) **Edit:** Jaime requested to merge this one first before he merges the database notebook branch)

**Review by weaverba137 — APPROVED** [review]

**Comment by weaverba137** [comment]
I found some additional problems, but they can be fixed quickly.

File: README.md (hunk 3 above)
Comment (stephjuneau) [reply README.md]: fixed.

File: getting_started/README.md (hunk 4 above)
Comment (stephjuneau) [reply getting_started/README.md]: fixed.

**Comment by stephjuneau** [comment]
OK, I made both new fixes and will merge. Thanks for the reviews!

**Pull request merged**